apps/ui/src/components/AuctionBidForm.vue

```diff
@@ -0,0 +1,255 @@
+    userBalance.value &&
+    amount > parseFloat(formattedBalance.value)
+  ) {
+    return 'Insufficient balance';
+  }
+  return undefined;
+});
+
+const priceError = computed(() => {
+  if (!bidPrice.value) return undefined;
+  const price = parseFloat(bidPrice.value);
+  if (price <= 0) return 'Invalid price';
+
+  const limit = showPriceInverted.value
+    ? 1 / clearingPrice.value
+    : clearingPrice.value;
+  const isInvalid = showPriceInverted.value ? price >= limit : price <= limit;
+
+  if (isInvalid) {
+    return `${showPriceInverted.value ? 'Maximum' : 'Minimum'} ${_n(limit)} ${priceLabel.value}`;
+  }
+  return undefined;
+});
+
+const canCancelOrder = computed(
+  () => parseInt(props.auction.orderCancellationEndDate) > Date.now() / 1000
+);
+
+const { data: userBalance, isError: isBalanceError } = useQuery({
+  queryKey: computed(() => [
+    'balance',
+    web3Account.value,
+    props.auction.addressBiddingToken
+  ]),
+  queryFn: async () => {
+    if (!web3Account.value) return '0';
+    return (
+      await new Contract(
+        props.auction.addressBiddingToken,
+        abis.erc20,
+        provider.value
+      ).balanceOf(web3Account.value)
+    ).toString();
+  },
+  enabled: computed(() => !!web3Account.value)
+});
+
+const { data: totalSupply, isError: isSupplyError } = useQuery({
+  queryKey: computed(() => ['supply', props.auction.addressAuctioningToken]),
+  queryFn: async () =>
+    (
+      await new Contract(
+        props.auction.addressAuctioningToken,
+        abis.erc20,
+        provider.value
+      ).totalSupply()
+    ).toString()
+});
+
+onMounted(() => {
+  if (clearingPrice.value > 0) {
+    bidPrice.value = (clearingPrice.value * DEFAULT_PRICE_PREMIUM).toFixed(
+      PRICE_DECIMALS
+    );
+  }
+});
+
+function togglePriceMode() {
+  const price = parseFloat(bidPrice.value);
+  if (price) {
+    bidPrice.value = (1 / price)
+      .toFixed(INVERTED_PRICE_DECIMALS)
+      .replace(/\.?0+$/, '');
+  }
+  showPriceInverted.value = !showPriceInverted.value;
+}
+</script>
+
+<template>
+  <div>
+    <h4 class="mb-3 eyebrow flex items-center gap-2">
+      <IH-cash />
+      Place bid
+    </h4>
+
+    <UiMessage
+      v-if="web3Account && (isBalanceError || isSupplyError)"
+      type="danger"
+      class="mb-3"
+    >
+      Failed to load balance or token supply. Please try again.
+    </UiMessage>
+
+    <div class="border border-skin-border rounded-lg">
+      <div class="px-4 py-4">
+        <div class="flex justify-between items-center mb-2 text-skin-text">
+          <div>Amount to bid</div>
+          <div v-if="web3Account && userBalance" class="text-sm">
+            Balance: {{ _n(parseFloat(formattedBalance)) }}
+            {{ props.auction.symbolBiddingToken }}
+          </div>
+        </div>
+        <div class="relative">
+          <UiInputAmount
+            v-model="bidAmount"
+            :definition="{ type: 'number', title: '', examples: ['0.0'] }"
```

Comment:
Can we use s-box for the whole form and use input label similar than other form, to have a style like this:

<img width="214" height="95" alt="Image" src="https://github.com/user-attachments/assets/8a87af0e-4e9b-4cd7-9d63-97d9ba30e96c" />